front/app/containers/InitiativesShow/LargerThanPhone.tsx

```diff
@@ -287,18 +282,19 @@ const LargerThanPhone = ({
                 <FileAttachments files={initiativeFiles.data} />
               </Box>
             )}
-            {hasOfficialFeedback && (
```

Comment:
Wasn't showing anymore when there were 0 feedback messages. So now the condition is only used for the styles (which was the initial goal).